services/security/security.go

```diff
@@ -163,12 +163,64 @@ const (
 type Operation string
 
 const (
-	ReadAccess   Operation = "read"
-	WriteAccess  Operation = "write"
-	DeleteAccess Operation = "delete"
-	AllAccess    Operation = "all"
+	OBSOLETE_ReadAccess   Operation = "read"
```

Comment:
Dowe know if they are being used outside this repo?

Reply:
Don't believe so. Only place I would think is moccli but I checked and they don't there.